src/FBGraphRequestManager.ts

```diff
@@ -17,24 +17,26 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * @flow
  * @format
  */
 
-'use strict';
+import {NativeModules} from 'react-native';
 
-const NativeGraphRequestManager =
-  require('react-native').NativeModules.FBGraphRequest;
+const NativeGraphRequestManager = NativeModules.FBGraphRequest;
 
-import type GraphRequest from './FBGraphRequest';
+import GraphRequest from './FBGraphRequest';
 
-type Callback = (error: ?Object, result: ?Object) => void;
+export type Callback = (
+  error?: Record<string, unknown>,
+  result?: Record<string, unknown>,
+) => void;
 
 function _verifyParameters(request: GraphRequest) {
-  if (request.config && request.config.parameters) {
-    for (var key in request.config.parameters) {
-      var param = request.config.parameters[key];
-      if (typeof param === 'object' && param.string) {
+  if (request.config?.parameters) {
+    for (const key in request.config.parameters) {
+      const param = request.config.parameters[key];
+
+      if (typeof param === 'object' && (param as any)?.string) {
```

Comment:
`(param as any)` causes the warning, but I currently cannot figure out the solution